frontend/src/layout/navigation-3000/navigationLogic.ts

```diff
@@ -153,12 +154,21 @@ export const navigation3000Logic = kea<navigation3000LogicType>([
             }
         },
     })),
-    events(({ actions, cache }) => ({
+    events(({ actions, values, cache }) => ({
         afterMount: () => {
             cache.onResize = () => actions.syncSidebarWidthWithViewport()
             cache.onKeyDown = (e: KeyboardEvent) => {
                 if (e.key === 'b' && (e.metaKey || e.ctrlKey)) {
                     actions.toggleSidebar()
+                    e.preventDefault()
+                }
+                if (e.key === 'f' && e.shiftKey && (e.metaKey || e.ctrlKey)) {
+                    const logic = values.activeNavbarItem.pointer.findMounted()
+                    if (logic && 'setIsSearchShown' in logic.actions) {
+                        logic.actions.setIsSearchShown(true)
+                        document.getElementById(SIDEBAR_SEARCH_INPUT_ID)?.focus()
```

Comment:
nit: I'd have expected this to use a ref instead of `getElementById`. Any specific reason why we use an id here? Fine to go in as is and no need to change if using ref needs to more props passing.

Reply:
Fair, made this a ref passed in via logic props